transcription/transcription_test.go

```diff
@@ -53,3 +55,36 @@ func TestSendEmailReturnsError(t *testing.T) {
 	err := SendEmail(username, password, host, port, to, subject, body)
 	assert.Error(err)
 }
+
+func TestDownloadFileFromURL(t *testing.T) {
```

Comment:
@sandlerben lmk what your research says. is the problem you were mentioning related to go 1.5, which the mocking library might still be running on?